.golangci.yaml

```diff
@@ -0,0 +1,114 @@
+linters:
```

Comment:
Is this the same config we have in CPMSO?

Reply:
Yeah pretty much the same